DigitalLearningSolutions.Web/Controllers/LearningMenuController/LearningMenuController.cs

```diff
@@ -36,6 +36,9 @@ public IActionResult Index(int customisationId)
                 return RedirectToAction("StatusCode", "LearningSolutions", new { code = 404 });
             }
 
+            var progressId = courseContentService.GetProgressId(User.GetCandidateId(), customisationId);
+            courseContentService.UpdateLoginCountAndDuration(progressId);

```

Comment:
Please add a test to the LearningMenuController checking that this method is called. We should actually have a test for the GetCourseContent method being called as well, would you mind adding that too? 🙂